test/library/draft/arrow/ArrowPrinting.chpl

```diff
@@ -0,0 +1,41 @@
+module ArrowPrinting {
+  require "ArrayDecl.chpl", "ArrowRecordDecl.chpl", "ParquetDecl.chpl";
+  private use ArrayDecl, ArrowRecordDecl, ParquetDecl;
+  use SysCTypes, CPtr;
+
+  // Functions for printing
+
+  proc printArray(array: GArrowArray) {
+    var error: GErrorPtr;
+    var str: c_string = garrow_array_to_string(array, c_ptrTo(error));
+    if(str == ""){
+      g_print("Failed to print: %s\n", error.deref().message);
+      g_error_free(error);
+      return;
+    }
+    g_print("%s\n",str);
+  }
+
+  proc printRecordBatch(recordBatch: GArrowRecordBatch) {
+    var error: GErrorPtr;
+    var str = garrow_record_batch_to_string(recordBatch, c_ptrTo(error));
+    if(str == ""){
+      g_print("Failed to print: %s\n", error.deref().message);
+      g_error_free(error);
+      return;
+    }
+    g_print("%s\n",str);
+  }
+
+  proc printTable(table: GArrowTable) {
+    var error: GErrorPtr;
+    var str = garrow_table_to_string(table, c_ptrTo(error));
+    if(str == ""){
+      g_print("Failed to print: %s\n", error.deref().message);
+      g_error_free(error);
+      return;
+    }
+    g_print("%s\n",str);
+  }
```

Comment:
Also, I wonder if these three functions could use the typeless arguments and then let the compiler decide which functions to generate since they are all identical aside from that single line. It could just be a function with no type (`proc printArrow(arrowValue)`) and then we check the type inside the function to determine which of the three to call. Just an idea.

Reply:
I haven't looked into this much but maybe it is possible to actually overload some function which converts any object to string so we can just use this with our native `write` or `writeln` functions